playwright-e2e/lib/component/Question.ts

```diff
@@ -24,19 +25,23 @@ export default class Question {
   }
 
   /**
-   * A dropdown.
-   * <br> Tag name: mat-select or select
-   * @param value
+   * Tag Select.
+   * <br> Note: use toSelect() for tag "mat-select".
+   * @param label
    */
-  select(value?: string): Locator {
```

Comment:
As I found out, `Select()` function does not work for tag `mat-select`. Change this function to support only tag `Select`, and new `toSelect()` function to support tag `mat-select`.